extern/storage-sealing/commit_batch.go

```diff
@@ -285,7 +286,14 @@ func (b *CommitBatcher) processBatch(cfg sealiface.Config) ([]sealiface.CommitBa
 		return []sealiface.CommitBatchRes{res}, xerrors.Errorf("couldn't get miner info: %w", err)
 	}
 
-	goodFunds := big.Add(b.feeCfg.MaxCommitGasFee, collateral)
+	bf, err := b.api.ChainBaseFee(b.mctx, tok)
+	if err != nil {
+		return []sealiface.CommitBatchRes{res}, xerrors.Errorf("couldn't get base fee: %w", err)
+	}
+
+	aggFee := miner5.AggregateNetworkFee(len(infos), bf)
```

Comment:
This should be added to the actor policy wrappers and abstracted, but I was lazy...Can do so if people feel strongly.

Reply:
I'd just do it. Otherwise, we'll miss it if it changes (and this is something I can definitely see changing.

Reply:
kk done